README.md

```diff
@@ -120,6 +120,15 @@ By default Phoenix imports `Ecto.Model` in the generated `ConnCase` and
 import Ecto.Model, except: [build: 2]
 ```
 
+If you want to keep the factories in the `test/factories` folder, change this line in mix.exs:
+
+```elixir
+# "test/factories" added the the paths array
```

Comment:
What do you think of changing this to `# Add "test/factories" to the end of the list`